### PR TITLE
filter_log_to_metrics: Add documentation for add_label

### PR DESCRIPTION
**Pull request by wndhydrnt**
Related changes in Fluent Bit:
- https://github.com/fluent/fluent-bit/pull/7739
- https://github.com/fluent/fluent-bit/pull/7967

### CODE REVIEW

**Comment by cosmo0920** [comment]
Could you resolve DCO error on the commit https://github.com/fluent/fluent-bit-docs/pull/1167/commits/8074ff6608a1be7df5434b6e9121b777f27eedbd?

File: pipeline/filters/log_to_metrics.md [hunk 1]
```diff
@@ -39,12 +40,12 @@ The following example takes records from two dummy inputs and counts all message
...
 [INPUT]
     Name               dummy
-    Dummy              {"message":"dummy", "kubernetes":{"namespace_name": "default", "docker_id": "abc123", "pod_name": "pod1", "container_name": "mycontainer", "pod_id": "def456"}, "duration": 20, "color": "red", "shape": "circle"}
+    Dummy              {"message":"dummy", "kubernetes":{"namespace_name": "default", "docker_id": "abc123", "pod_name": "pod1", "container_name": "mycontainer", "pod_id": "def456", "labels":{"app": "app1"}}, "duration": 20, "color": "red", "shape": "circle"}
     Tag                dummy.log
 
 [INPUT]
     Name               dummy
-    Dummy              {"message":"hello", "kubernetes":{"namespace_name": "default", "docker_id": "abc123", "pod_name": "pod1", "container_name": "mycontainer", "pod_id": "def456"}, "duration": 60, "color": "blue", "shape": "square"}
+    Dummy              {"message":"hello", "kubernetes":{"namespace_name": "default", "docker_id": "abc123", "pod_name": "pod1", "container_name": "mycontainer", "pod_id": "def456" "labels":{"app": "app1"}}, "duration": 60, "color": "blue", "shape": "square"}
```
Comment (cosmo0920) [inline pipeline/filters/log_to_metrics.md:48]: It seems that a missing "," is between  "def456" and "labels". Without it, the change makes invalid JSON parameter.

**Review by cosmo0920 — CHANGES_REQUESTED** [review]
Adding `add_label` parameter looks great. I found a invalid JSON parameter. Could you take a look then?

File: pipeline/filters/log_to_metrics.md (hunk 1 above)
Comment (wndhydrnt) [reply pipeline/filters/log_to_metrics.md]: Fixed in df180040a4996e9aacc7be53c04de91d056c17fc.

**Comment by wndhydrnt** [comment]
Hello @cosmo0920 @lecaros 

Fluent Bit 2.10.0, which contains the `add_label` feature, has been released a few weeks ago already. This PR is still open so people have no way to discover through the documentation that the feature exists.

What is needed to get this PR merged?

**Review by cosmo0920 — APPROVED** [review]

**Comment by lecaros** [comment]
Hi @wndhydrnt,
Are we okay to include this doc without having https://github.com/fluent/fluent-bit/pull/7967 merged yet?

**Pull request merged**